cc_corpus/deduplication.py

```diff
@@ -49,7 +49,8 @@ def write_results(self, input_file, results):
             self.mh_offset += self.minhashf.write(pickle.dumps(mh))
         for id_fields in results['id']:
             self.di_offset += self.doc_idf.write(
-                '{}\n'.format('\t'.join(str(f) for f in id_fields)).encode('utf-8'))
+                '{}\n'.format('\t'.join(str(f) for f in id_fields))
+                .encode('utf-8'))
```

Comment:
The closing `)` should be on a separate line, as per PEP-8, in the same column as the `s` in `self`.